### ISSUE TITLE
"Click here" is a poor link text

### ISSUE THREAD

**Issue by gunnarvelle** [NodeBB/NodeBB#13020]
### Description

In the help dialog, the link to markdown documentation should be the full sentence giving more context to screen readers, not only click here.
![image](https://github.com/user-attachments/assets/b0add3f0-3bd9-4b0d-8c05-2ff0de43bf82)
